x-pack/legacy/plugins/siem/public/store/network/model.ts

```diff
@@ -19,6 +19,8 @@ export enum NetworkType {
 }
 
 export enum NetworkTableType {
+  alerts = 'alerts',
+  anomalies = 'anomalies',
```

Comment:
I don't think the anomalies table stores any data in redux -- we may want to remove this to avoid any confusion.